src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -175,6 +175,41 @@ uct_ib_mlx5_inline_copy(void *restrict dest, const void *restrict src, unsigned
 }
 
 
+/**
+ * Copy uct_iov_t array to inline segment, taking into account QP wrap-around.
+ *
+ * @param dest     Inline data in the WQE to copy to.
+ * @param iov      A pointer to an array of uct_iov_t elements.
+ * @param iov_cnt  A number of elements in a iov array.
+ * @param length   Total data length.
+ * @param wq       Send work-queue
```

Comment:
minor: better add `.` in the end for consistency with other lines in this comment